tests/Feature/Nova/OrderResourceTest.php

```diff
@@ -24,5 +31,42 @@ public function index()
             ->assertOk();
 
         $this->assertCount(4, $response->json('resources'));
+
+        $orderFields = collect($response->json('resources')[0]['fields'])->pluck('attribute');
+
+        $this->assertCount(4, $orderFields);
+        $this->assertEquals([
+            'id',
+            'order_number',
+            'amount',
+            'created_at',
+        ], $orderFields->toArray());
+    }
+
+    /** @test */
+    public function index_with_stub_resources()
+    {
+        $this->createStubNovaResources();
+
+        Order::factory()->count(4)->create();
+
+        $this->actingAs(User::factory()->create());
+
+        $response = $this->getJson('nova-api/orders')
+            ->assertOk();
+
+        $this->assertCount(4, $response->json('resources'));
+
+        $orderFields = collect($response->json('resources')[0]['fields'])->pluck('attribute');
+
+        $this->assertCount(6, $orderFields);
+        $this->assertEquals([
+            'id',
+            'order_number',
+            'customer',
+            'location',
+            'amount',
+            'created_at',
```

Comment:
With stub resources created, customer and location are in the field set returned.